source/Account.cpp

```diff
@@ -341,17 +349,10 @@ void Account::AddFine(int64_t amount)
 
 
 
-// Death benefits have a short term but lower interest than the best mortgage rate.
-void Account::AddDeathBenefits(int64_t reparations)
-{
-	mortgages.emplace_back(reparations, 1000, 60);
-}
-
-
-// Unpaid profit shares have a short term and a high but variable interest rate.
+// Unpaid profit shares have a medium term and a low interest rate.
```

Comment:
I think a higher interest rate would be better, to encourage the player not to let them hang around for too long.

Reply:
Actually, the interest rate and term need to be defined by a data files. Maybe the game start settings?

Reply:
I agree, but Kiko wanted *no* interest, so I thought this was good compromise?

Reply:
Let's let the data files decide. 😄

Reply:
Sure, though I don't know how to do that yet.  Also as you can see in Mortgage.cpp below, the game determines what to call a debt based on its interest rate.  Not an ideal way of doing it, I know, but if you let the data change this then how that works will have to be changed as well.

Reply:
```suggestion
// Shared profits that the player owes the fleet, but hasn't paid yet.
```